azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/reducer/ParticipantsReducer.kt

```diff
@@ -40,7 +41,7 @@ internal class ParticipantsReducerImpl : ParticipantsReducer {
 
                 var updatedState = state
 
-                if (action.localParticipantRemoved) {
+                if (hasLocalParticipant) {
```

Comment:
Before this if was based on the local participant being removed but now it's based on having the local participant?

Reply:
right, the naming might be confusing, but `localParticipantRemoved` is a boolean variable inside the action. since `hasLocalParticipant` is within the scope of `is ParticipantAction.ParticipantsRemoved { ... } ` so it serves the same meaning syntactically speaking.

Reply:
sorry not sure if I fully understand your question, but please let me know if you need more clarifications